syntax = "proto2";

option go_package = "github.com/ydb-platform/nbs/cloud/blockstore/tools/common/go/cms/proto;kikimr";

package NKikimrConfig;

message TNamedConfig {
    optional string Name = 1;
    optional bytes Config = 2;
}

message TAppConfig {
    repeated TNamedConfig NamedConfigs = 100;
}
